web/cypress/support/page_objects/onboardingPage.ts

```diff
@@ -0,0 +1,72 @@
+import { LookupIndustryById } from "@businessnjgovnavigator/shared/";
+
+export class OnboardingPage {
+  getHasExistingBusiness(radio: boolean) {
+    return cy.get(`input[name="has-existing-business"][value="${radio}"]`);
+  }
+
+  getBusinessName() {
+    return cy.get("#businessName");
+  }
+
+  getIndustryDropdown() {
+    return cy.get('[data-testid="industryid"]');
+  }
+
+  getLegalStructureRadio(companyType: string) {
+    return cy.get(`[data-value="${companyType}"]`);
+  }
+
+  getLiquorLicense(radio?: boolean) {
+    return cy.get(`input[name="liquor-license"]${radio === undefined ? "" : `[value="${radio}"]`}`);
+  }
+
+  getHomeBased(radio?: boolean) {
+    return cy.get(`input[name="home-based-business"]${radio === undefined ? "" : `[value="${radio}"]`}`);
+  }
+
+  getLocationDropdown() {
+    return cy.get('[data-testid="municipality"]');
+  }
+
+  getNextButton() {
+    return cy.get('[data-testid="next"]');
```

Comment:
does this need to be its own function? if we're only ever using it to click the next button?